Tests.Unit.Core.ApplicationServices/ApplicationServices/UserServiceTest.cs

```diff
@@ -249,7 +251,7 @@ public void GetUserInOrganization_Returns_NotFound()
 
 
         [Fact]
-        public void DeleteUserFromKitos_Raises_EntityBeingDeletedEvent_And_AccessRights_Changed()
+        public void DeleteUser_Raises_EntityBeingDeletedEvent_And_AccessRights_Changed()
```

Comment:
think it is time to introduce commands into the kitos domain.. this cannot continue. Looking at the code it makes no sense that it raises an "event" = something that happened if what it means is "Do this!".. I will introduce it